resolwe_bio/tests/files/cuffnorm_all_fpkm_means.txt

```diff
@@ -1,5 +1,5 @@
-Group1	Group3	Group2
-CSNK2A1	114564.5	118186	113357.333333333
-RBCK1	117094.1	140182	109398.133333333
-TBC1D20	41356.25	36121.1	43101.3
-TRIB3	121871.75	89448.5	132679.5
+Group1	Group2	Group3
```

Comment:
Don't change the order it's an extra test to handle such case.

Reply:
Before `replicates = set(args.replicates.split(","))` was changed to `replicates = sorted(list(set(args.replicates.split(","))))` this order was different every time the test ran. Now it is the same every time.